server/service/src/dashboard/item_count.rs

```diff
@@ -88,11 +117,40 @@ impl ItemCountServiceTrait for ItemServiceCount {
         let more_than_six_months_stock =
             Self::get_more_than_six_months_stock_count(&self, &item_stats);
 
+        let stock_risk_window_months =
```

Comment:
Pref have such a long name, still debating whats a good short name for this

Reply:
Would nit and just put num_months_consumption and below for format can do:

msg: format!("Failed to load preference: {e}"),